frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/ticketForm.component.tsx

```diff
@@ -32,9 +32,15 @@ interface ModulePanelProps {
 
 const ModulePanel = ({ module, scrollPanelIntoView, defaultExpanded, ...rest }: ModulePanelProps) => {
 	const required = some(module.properties, 'required');
+	const { title, ...accordionProps } = getModulePanelProps(module);
 	return (
-		// eslint-disable-next-line max-len
-		<Accordion {...getModulePanelTitle(module)} onChange={scrollPanelIntoView} defaultExpanded={defaultExpanded} required={required}>
+		<Accordion
```

Comment:
why this change?

Reply:
We want to indicate when a module has at least a required field. This was previously achieved passing a "required" property that added a red asterisk next the title.

However, this forced the accordion header to be a `FormLabel`, which was bad as, on click (expand/collapse accordion), a checkbox inside it would also get the click, and so toggles its state.

This change was to make the header a span, so it's down to the parent to decide what the title should look like, without affecting the rest of the content
(I have problems posting pictures, but checkout [frontend/src/v5/ui/controls/accordion/accordion.component.tsx](https://github.com/3drepo/3drepo.io/pull/4196/files/db1d82ac58fcf5619c69b4bd6c348612de8c8f74#diff-85f1acf0bb813aba2edad551be532ac7690ad44123825dc11d17da5df75831c1) and [frontend/src/v5/ui/controls/accordion/accordion.styles.tsx](https://github.com/3drepo/3drepo.io/pull/4196/files/db1d82ac58fcf5619c69b4bd6c348612de8c8f74#diff-138913dbc3a8af50b5bb822b9cb4cbaeb6bd4975dabf6be8bfc48f7b892537b7))

P.S. I forgot to remove the prop required which is still being passed to the Accordion for no reason